api/project.godot

```diff
@@ -0,0 +1,23 @@
+; Engine configuration file.
```

Comment:
Can this project file also be moved to test? My thinking is that when developers clone this code, they won't need this project file, so its better to put it in the test directory with the other files they can safely ignore.

Reply:
This file is required at the root directory of a godot project, and without it godot engine won't be able to open the project. 
My idea is that when we generate the release version we only take a subset of files from the /api directory, only releasing the ones needed.